etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
+PACHCTL=${PACHCTL:-pachctl}
+RM=${RM:-rm}
+SLEEP=${SLEEP:-sleep}
```

Comment:
Definitely does not need to be overridden

Reply:
sleep has different behavior in the gnu utilities than in macOS.  It allows arguments in the form of different units, including minutes and hours, vs the seconds-only approach of the macOS version.